src/Plutarch/Extra/DebuggableScript.hs

```diff
@@ -0,0 +1,211 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedRecordDot #-}
+{-# LANGUAGE RankNTypes #-}
+
+-- | @since 3.0.2
```

Comment:
Modules themselves don't usually have `since`s: my comment was a shorthand for 'everything exported in here'.

Reply:
Ah. Also need a line break between some @sinces...